openedx/stanford/djangoapps/register_cme/tests/test_form.py

```diff
@@ -151,3 +151,31 @@ def test_export_countries(self, country):
         form = ExtraInfoForm(data)
         is_valid = form.is_valid()
         self.assertFalse(is_valid)
+
+    @data(None, 'ab', 'longsunet')
+    def test_affiliation_sunet_id(self, sunet_id):
+        data = self.data
+        data['affiliation'] = 'Stanford University'
+        data['stanford_department'] = 'Medicine'
+        data['sunet_id'] = sunet_id
+        form = ExtraInfoForm(data)
+        is_valid = form.is_valid()
+        self.assertFalse(is_valid)
+
+    @data(None, 'invalid_department')
+    def test_affiliation_stanford_department(self, stanford_department):
+        data = self.data
+        data['affiliation'] = 'Stanford University'
+        data['stanford_department'] = stanford_department
+        data['sunet_id'] = 'sunet_id'
+        form = ExtraInfoForm(data)
+        is_valid = form.is_valid()
+        self.assertFalse(is_valid)
+
+    @data('1')
```

Comment:
It seems like `clean_postal_code` in the form only enforces a minimum length, so the former option seems fine?